app/services/calculate_standards.rb

```diff
@@ -1,33 +1,28 @@
 class CalculateStandards < ApplicationService
+  MANS_METABOLIC_MULTI = { common: 88.36, weight: 13.4, height: 4.8, age: 5.7 }
+  WOMANS_METABOLIC_MULTI = { common: 447.6, weight: 9.2, height: 3.1, age: 4.3 }
   MODIFIERS = { 1 => 1.2, 2 => 1.375, 3 => 1.725, 4 => 1.9 }.freeze
 
-  attr_reader age, sex, weight, height, activity
+  attr_reader :age, :sex, :weight, :height, :activity
 
   def initialize(profile)
     @profile = profile
-    @age = @profile.age
-    @sex = @profile.sex
-    @weight = @profile.weigh
-    @height = @profile.height
-    @activity_multiplier = MODIFIERS[@profile.activity]
+    age = @profile.age
+    sex = @profile.sex
+    weight = @profile.weigh
+    height = @profile.height
+    activity = MODIFIERS[@profile.activity]
   end
 
   def call
-    if @sex == 1
-      @profile.update(BMR: mans_metabolic_rate.round)
-      Success(@profile)
-    else
-      @profile.update(BMR: womens_metabolic_rate.round)
-      Success(@profile)
-    end
+    rates = sex == 1 ? MANS_METABOLIC_MULTI : WOMANS_METABOLIC_MULTI
+    @profile.update(BMR: calculate(rates).round)
```

Comment:
You use `rates` variable that can be calculated without implicit passing to a method.
Extract it to a separate method and call it right from `calculate` method.